### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by keithk**
Read the notes in [this ticket](https://app.clubhouse.io/glitch/story/20653/get-circleci-working-with-deployments-with-new-design-system) for this change! Our PRs now have a canary version too, which you can see attached to this PR!

- Update CONTRIBUTING.md to reflect versioning and pushing to the package repository is handled by a github action
- Update readme with details for installing from the github package repository 

<details>
  <summary>📦 Published PR as canary version: <code>0.19.2-canary.129.5cb491f.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @glitchdotcom/shared-components@0.19.2-canary.129.5cb491f.0
  # or 
  yarn add @glitchdotcom/shared-components@0.19.2-canary.129.5cb491f.0
  ```
</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
-This package has a peer dependency on React 16.8+ ("the one with hooks"); it also includes dependencies on prop-types and styled-components.
+This package is installed in the github packages repo, so you have to do a little
+configuration to set up your NPM to download the design system.
+
+In your project, ensure there is a `.npmrc` file containing:
+`registry=https://npm.pkg.github.com/glitchdotcom`
+
+Generate an [access token](https://github.com/settings/tokens) with the permissions `repo`, `write:packages` and `read:packages`.
+
+(TODO: save this access token to your dotfiles for cutting canary releases)
```
Comment (anjapatel) [inline README.md:14]: Is this meant to be in here as a 'TODO' ?

**Review by anjapatel — APPROVED** [review]
Looks great! Are there any special instructions for pulling in updates?

File: README.md (hunk 1 above)
Comment (keithk) [reply README.md]: Took that out, since it automatically builds canary releases!

**Comment by keithk** [comment]
:rocket: PR was released in [`v0.20.0`](https://github.com/glitchdotcom/shared-components/releases/tag/v0.20.0) :rocket:

**Pull request merged**